app/RquestBridge/Config/RQuestOptions.cs

```diff
@@ -0,0 +1,11 @@
+namespace RquestBridge.Config;
+
+public class RQuestOptions
+{
+  public int Id { get; set; }
```

Comment:
what is this ID, is it just left over from ActivitySource or is it actually meaningful?

Reply:
I left it in since I thought we might want to have one still to link to the job coming in from rquest

Reply:
sure, but that wouldn't be here in the application's config? otherwise you'd only ever have 1 job id?

Reply:
cool,I removed the "activitySource" id from the RquestJob as well. We still save the rquest uuid so we can use that jobId